src/openslide-vendor-dicom.c

```diff
@@ -711,6 +726,76 @@ static void add_properties(openslide_t *osr, struct dicom_level *level0) {
   }
 }
 
+static bool quickhash_level_pixels(struct _openslide_hash *hash,
+                                   struct dicom_level *level) {
+  double d;
+  if (!get_tag_decimal_str(level->file->metadata, NumberOfFrames, 0, &d)) {
+    return false;
```

Comment:
Are there slides where the lowest-resolution level won't have any frames, and we want to skip the quickhash but not fail the open?  It seems like we'd want to fail in that case.

Reply:
We'd need to give `quickhash_level_pixels()` three exit states: no suitable level (it's too big to hash quickly, I think this could happen), some kind of bad low level failure, and success, which seems awkward.

Putting the first two cases into one and making it disable quickhash seems like a useful simplification to me.

Reply:
... I made `quickhash_level_pixels()` into a standard GError-style function and moved the quickhash disable into it on the "level too big" path. Open will now fail during quickhash computation for missing NumberOfFrames, failed frame open, etc.